### DISCUSSION TITLE
查询策略 read_consistency_level 是全局配置，能否为单独的查询设置自己的查询策略来减少 leader 的压力

### DISCUSSION THREAD

**Discussion by wenmin864** [apache/iotdb#13639] [Q&A]
目前环境 3C3D 数据库 数据分区的leader 都在同一个节点上，多个查询会对节点造成很大压力 ，查询策略为全局配置，追求强一致性，是否可以为 单独查询设置查询策略

**Comment by CritasWang** [comment]
副本数配置的多少呢，leader 会自动均衡的

**Reply by wenmin864** [reply]
数据3 副本  使用 iot协议，目前看重启多次后  一个数据库的数据分区leader 在一个节点上，这个库是通过load tsfile重入的

**Comment by CritasWang** [comment]
可以贴一下 show regions 的截图。也可以直接在群里交流
